### PR TITLE
Fixed file modified time not changing on windows

### PR DESCRIPTION
**Pull request by darstahl**
Signed-off-by: Darren Stahl darst@microsoft.com

This fixes the lack of changing modified times on Windows. @jstarks @jhowardmsft

### CODE REVIEW

**Comment by calavera** [comment]
LGTM

**Comment by cpuguy83** [comment]
huh... os.Chtimes is under a posix file... didn't realize `posix` stuff was compiled on Windows.

**Comment by cpuguy83** [comment]
LGTM

**Comment by cpuguy83** [comment]
Getting another set of eyeballs on this just to make sure we aren't missing something.

File: builder/internals.go [hunk 1]
```diff
@@ -337,23 +336,17 @@ func calcCopyInfo(b *builder, cmdName string, cInfos *[]*copyInfo, origPath stri
 
 		// Set the mtime to the Last-Modified header value if present
 		// Otherwise just remove atime and mtime
-		times := make([]syscall.Timespec, 2)
+		mTime := time.Time{}
 
 		lastMod := resp.Header.Get("Last-Modified")
 		if lastMod != "" {
-			mTime, err := http.ParseTime(lastMod)
-			// If we can't parse it then just let it default to 'zero'
```
Comment (tiborvass) [inline builder/internals.go]: keep the comment please

Comment (darrenstahlmsft) [inline builder/internals.go]: will do

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**